cristin-project/src/main/java/no/unit/nva/cristin/projects/model/cristin/adapter/CristinPersonToPerson.java

```diff
@@ -19,15 +20,20 @@ public Person apply(CristinPerson cristinPerson) {
 
     private Person convert(CristinPerson cristinPerson) {
         return new Person(
-            convertId(cristinPerson.cristinPersonId()),
+            convertId(cristinPerson.identifiedCristinPerson(), cristinPerson.cristinPersonId()),
             cristinPerson.firstName(),
             cristinPerson.surname(),
             cristinPerson.email(),
             cristinPerson.phone());
     }
 
-    private URI convertId(String cristinPersonId) {
-        return getNvaApiId(cristinPersonId, PERSON_PATH_NVA);
+    private URI convertId(Boolean identified, String cristinPersonId) {
+        return identifiedFieldNotPresent(identified) || identified
```

Comment:
So, if we don't have the data, or the person is identifier…we keep the URI?

Reply:
Exactly. We dont want to remove the id if the field is missing. This makes it backwards compatible and if we want to enforce the field later it is a simple change in proxy